### PR TITLE
Clean-up Actuator Line and implement epsilon/chord scaling.

### PR DESCRIPTION
**Pull request by tonyinme**
This is a small improvement to the actuator line code.
The code has been changed, cleaned up and documentation for the feature has been added.
The actuator point drag info files have been removed.

The examples section has been edited and some bugs have been fixed.
The files in the examples have been organized in a more consistent way within directories instead of having many files.

### CODE REVIEW

**Comment by sayerhs** [comment]
@tonyinme You removed the `actuatorLine` test files, but it appears that you didn't remove the `add_test` line in [CTestList.cmake](https://github.com/tonyinme/nalu-wind/blob/master/reg_tests/CTestList.cmake#L79). Can you update this? Or else nightly tests will fail when it attempts to run a non-existent test.

**Comment by gantech** [comment]
@tonyinme It looks like too many files have changed with just whitespace changes. I'm having a hard time figuring out what algorithm actually changed. Could you create a clean fork off the latest master with just one commit and change only the files where the algorithm changed? 

And regarding the changes in the examples... could you give a brief overview of what changed?

**Comment by tonyinme** [comment]
hey @gantech, the reason for the white spaces is that I ran a formatter on the code. That took some of the files and indented them in a more consistent manner.
Most of the ALM algorithm changes are here: 
https://github.com/Exawind/nalu-wind/pull/183/commits/47b5db56f71669c873a38604cb21e6663fa091f6

The changes in the example are minor, and I will submit a pull request later on with many fixes.

**Comment by alanw0** [comment]
I would ask you not to re-format the code by pushing the results of an automatic formatter.

**Comment by tonyinme** [comment]
Hey all, I apologize for the confusion regarding the spacing. I prefer to use 4 spaces to indent, so that is what I was using to improve the readability of the code. I have now changed the spacing in all files back to 2 spaces to be consistent with the nalu-wind format. 
This formatting only affected the files I was working on, and all other files in the code-based have not been modified.
@alanw0, I really appreciate the consistency and clean-look of a piece of code after going through an automated formatter, but if everyone agrees that this should not be used, I will not use it again.  
We could also consider running the formatter only once on every file and use 4 spaces as default indentation? haha

**Comment by alanw0** [comment]
I don't have a strong preference on depth of indenting etc. My main problem with auto-formatting the code-base is, it messes up git history and causes conflicts for anyone who has un-pushed changes.

**Comment by sayerhs** [comment]
@tonyinme running a formatter is undesirable as it removes the ability to track history through git commits using `git blame`. You'll have to get several commits prior to see who was responsible for coding up some specific part of the code. For this reason, it would have been better if you had removed the commit that introduced formatting changes instead of another commit that just changed the formatting changes.

File: src/ActuatorDiskFAST.C [hunk 1]
```diff
@@ -226,12 +226,12 @@ ActuatorDiskFAST::add_swept_points_to_map()
...
-              np,
-              make_unique<ActuatorFASTPointInfo>(
-                iTurb, centroidCoords, searchRadius,
-                actuatorInfo->epsilon_, // TODO(psakiev)::scale epsilon based on
-                                        // the arc length ratio with the tip
-                fast::BLADE, i)));
+                                           np,
+                                           make_unique<ActuatorFASTPointInfo>(
+                                             iTurb, centroidCoords, searchRadius,
+                                             actuatorInfo->epsilon_min_, // TODO(psakiev)::scale epsilon based on
```
Comment (psakievich) [inline src/ActuatorDiskFAST.C]: You can delete this TODO comment.  The chord scaling gives the desired behavior

**Comment by tonyinme** [comment]
I have added the epsilon in chord, thickness and spanwise directions following the suggestions from @gantech @sayerhs and @mchurchf . This capability is now on-par with SOWFA. The documentation has instructions for how to use this. In the case of not specifying the epsilon_chord variables, the default behavior is used. I also merged the latest master to avoid some conflicts.

**Comment by gantech** [comment]
Hey @tonyinme . This is looking good. Did you update the regression test to use the anisotropic spreading? I think it's a good idea because that will ensure that we catch any bugs in this new piece of code going forward.

**Comment by tonyinme** [comment]
Hey all, 
Thank for the feedback. I believe this is ready to merge.
@gantech, The regression test is the same as it was before (the changes in this pull request did not affect it). I will work on updating the regression test or adding a different one once I am back from holiday.
@michaelasprague, feel free to merge this.
Thanks!
Tony

**Pull request merged**